x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/TokenService.java

```diff
@@ -499,19 +507,25 @@ public void invalidateAccessToken(String tokenString, ActionListener<Boolean> li
      *
      * @see #invalidateAccessToken(String, ActionListener)
      */
-    public void invalidateAccessToken(UserToken userToken, ActionListener<Boolean> listener) {
+    public void invalidateAccessToken(UserToken userToken, ActionListener<TokensInvalidationResult> listener) {
         ensureEnabled();
         if (userToken == null) {
             logger.trace("No access token provided");
             listener.onFailure(new IllegalArgumentException("token must be provided"));
         } else {
             maybeStartTokenRemover();
             final long expirationEpochMilli = getExpirationTime().toEpochMilli();
-            indexBwcInvalidation(userToken, listener, new AtomicInteger(0), expirationEpochMilli);
+            indexBwcInvalidation(Collections.singleton(userToken.getId()), listener, new AtomicInteger(0), expirationEpochMilli, null);
```

Comment:
I'm leaving this here to link the issue, but I'd love to drop this for 7.0 in a followup https://github.com/elastic/elasticsearch/issues/36727. If you want to do it that's fine; if not someone else can pick it up

Reply:
I'll happily pick this up :)